test/routes/viewModels/taskListModelsCA.test.js

```diff
@@ -909,8 +921,131 @@ describe('TaskList models', () => {
         reviewCase,
         postponeOrRefuse,
         refuse,
+        resubmit,
+        createLicence,
+      ])
+    })
+
+    test('should return list of tasks for standard route excluding resubmit to DM', () => {
+      expect(
+        taskListModel(
+          'CA',
+          false,
+          {
+            decisions: {
+              eligible: true,
+              curfewAddressApproved: true,
+              bassReferralNeeded: false,
+              bassWithdrawn: false,
+              bassExcluded: false,
+              bassAccepted: null,
+              optedOut: false,
+              dmRefused: undefined,
+              excluded: false,
+            },
+            tasks: {
+              bassAreaCheck: 'UNSTARTED',
+              bassOffer: 'UNSTARTED',
+            },
+            stage: 'DECIDED',
+          },
+          {},
+          null
+        )
+      ).toEqual([
+        eligibilitySummary,
+        proposedCurfewAddressEdit,
+        riskManagement,
+        victimLiasion,
+        curfewHours,
+        additionalConditionsEdit,
+        reportingInstructions,
+        reviewCase,
+        postponeOrRefuse,
+        refuse,
         createLicence,
       ])
     })
+
+    test('should return list of tasks for standard route INCLUDING resubmit but not Postpone or Refuse', () => {
+      expect(
+        taskListModel(
+          'CA',
+          false,
+          {
+            decisions: {
+              eligible: true,
+              curfewAddressApproved: true,
+              bassReferralNeeded: false,
+              bassWithdrawn: false,
+              bassExcluded: false,
+              bassAccepted: null,
+              optedOut: false,
+              dmRefused: true,
+              excluded: false,
+            },
+            tasks: {
+              bassAreaCheck: 'UNSTARTED',
+              bassOffer: 'UNSTARTED',
+            },
+            stage: 'DECIDED',
+          },
+          {},
+          null
+        )
+      ).toEqual([
+        eligibilitySummary,
+        proposedCurfewAddressEdit,
+        riskManagement,
+        victimLiasion,
+        curfewHours,
+        additionalConditionsEdit,
+        reportingInstructions,
+        reviewCase,
+        resubmit,
+      ])
+    })
+  })
+
+  test('should return list of tasks for standard route INCLUDING resubmit AND Postpone or Refuse', () => {
+    expect(
+      taskListModel(
+        'CA',
+        false,
+        {
```

Comment:
Could maybe have a helper method that creates this object object for testing? It looks like this object only differs by its value of `decisions.dmRefused` so it would just help show what's changing.  So a bit like (aside from the method name, I'm not sure what that object is called):

```
const createObjectWithDmRefused = dmRefused => {
...
}
```

to give: 
```
expect(
      taskListModel(
        'CA',
        false,
        createObjectWithDmRefused(false),
        {},
        null
```